Jenkinsfile

```diff
@@ -191,6 +191,11 @@ pipeline {
                         deleteDir()
                     }
                 }
+
+                build job: 'WinContrail/gather-build-stats',
```

Comment:
Please add `wait: false`.
There will be a race condition. It should be fixed in new PR.